bigquery/storage/managedwriter/client.go

```diff
@@ -27,7 +27,6 @@ import (
 	"google.golang.org/api/option"
 	storagepb "google.golang.org/genproto/googleapis/cloud/bigquery/storage/v1"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/metadata"
```

Comment:
add a e2e just to be sure?

Reply:
the existing e2es already test in non-default regions.